File: mvc1/Cpp/16968 차량 번호판 1.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    string s;
    cin >> s;
    stack<int> st;
    int answer = 1;
    for(int i = 0 ; i < s.size(); i++) {
        if(st.empty()) {
            if(s[i] == 'd') st.push(10);
            else st.push(26);
        }
        
        else if(s[i] == 'd') {
            if(st.top() == 10 || st.top() == 9) st.push(9);
            else st.push(10);
        }
        else if(s[i] == 'c') {
            if(st.top() == 26 || st.top() == 25) st.push(25);
            else st.push(26);
        }
    }
    while(!st.empty()) {
        answer = answer * st.top();
        st.pop();
    }
    cout << answer;
}

File: mvc1/Python/16968 차량 번호판 1.py
s = input(); answer = 10 if s[0] == 'd' else 26
for i in range(1, len(s)) :
    if s[i] == 'd' :
        if s[i] ==  s[i-1] : answer *= 9
        else : answer *= 10
    else :
        if s[i] == s[i-1] : answer *= 25
        else : answer *= 26
print(answer)
